demos/accessible-pieChart.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```

Comment:
As discussed, this demo for accessibility testing can be removed from the pull request.